Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -113,28 +113,28 @@ final class LayerTests: XCTestCase {
     func testConv2DGradient() {
         let filter =  Tensor(shape: [3, 3, 2, 4], scalars: (0..<72).map(Float.init))
         let bias = Tensor<Float>(zeros: [4])
-        let layer = Conv2D<Float>(filter: filter, 
-                                  bias: bias, 
+        let layer = Conv2D<Float>(filter: filter,
+                                  bias: bias,
                                   activation: identity,
-                                  strides: (2, 2), 
+                                  strides: (2, 2),
                                   padding: .valid)
         let input = Tensor(shape: [2, 4, 4, 2], scalars: (0..<64).map(Float.init))
         let grads = gradient( at: input, layer) { $1($0).sum() }
-        // The expected gradients were computed using the following Python code: 
+        // The expected gradients were computed using the following Python code:
```

Comment:
#499

Reply:
This is already taken care of by #502, I'll pull into master.